ablator/modules/storage/cloud.py

```diff
@@ -80,8 +81,8 @@ def _make_cmd_up(self, local_path: Path, destination: str):
         list[str]
             The command to upload the file.
         """
-
-        destination = str(Path(self.bucket) / destination / local_path.name)
+        local_path_striped = str(local_path).lstrip("/")
```

Comment:
error prone for windows. Better to use Path.

Reply:
I found a more elegant way of doing this, now the code can handle both windows and posix path, I am testing it right now.

Reply:
fixed, I found a better way of dealing with path in different operation systems.